st2common/tests/unit/test_jinja_render_regex_filters.py

```diff
@@ -68,3 +68,24 @@ def test_filters_regex_search(self):
         actual = env.from_string(template).render({'k1': 'v0.10.1'})
         expected = 'True'
         self.assertEqual(actual, expected)
+
+    def test_filters_regex_substring(self):
+        env = jinja_utils.get_jinja_environment()
+
+        # Normal (match)
+        template = '{{k1 | regex_substring("(127.0.0.1)")}}'
```

Comment:
I assume user can also user other regex modifiers such as `\w+`, `.*`, etc. in the substring, right?

Otherwise functionality makes little sense since it just returns an original static sub-string as passed in.

If that's the case, it would also be good to add a test case with a more "realistic" scenario which uses `.*` or similar